.env.template

```
@@ -207,6 +207,14 @@ REACT_APP_AUTH0_REDIRECT_URI=http://localhost:3000/auth/callback
 REACT_APP_AUTH0_SCOPE="openid profile email"
 REACT_APP_AUTH0_AUDIENCE=https://bff.vaspdirectory.net
 REACT_APP_SENTRY_DSN=
+REACT_APP_SENTRY_ENVIRONMENT=
+
```

@bbengfort Do we need to add the following environment variables to .env.template too? They're in [`directory/web/gds-user-ui/.env.example`](https://github.com/trisacrypto/directory/blob/2a5f83432363e7e420fec79f3e7a36ec328ddedb/web/gds-user-ui/.env.example)

```suggestion
REACT_APP_GIT_REVISION=
REACT_APP_USE_DASH_LOCALE=
```

---

Good catch!